gradle/pre-merge.sh

```diff
@@ -1,26 +1,50 @@
 #!/bin/bash -e
 BUILD_NAME=${GITHUB_RUN_ID:=local-$(date +%s)}
 
-# Get Launchable CLI installed. If you can, make it a part of the builder image to speed things up
+echo
+echo "===== BEGIN LAUNCHABLE ====="
+
+echo
+echo "# Install the Launchable CLI. If you can, install it as part of the builder image to speed things up"
+echo "# Command: pip3 install --user launchable~=1.0 > /dev/null"
+echo
 pip3 install --user launchable~=1.0 > /dev/null
+
 export PATH=~/.local/bin:$PATH
 
-set -x
-# Verify that Launchable setup is all correct. Useful primarily while you work on integration
+echo "# Verify that Launchable setup is all correct. Useful primarily while you work on integration"
+echo "# Command: launchable verify"
+echo
 launchable verify
 
-# Tell Launchable about the build you are producing and testing
+echo
+echo "# Tell Launchable about the build you are producing and testing"
+echo "# Command: launchable record build --name \"\$BUILD_NAME\" --source .."
+echo
 launchable record build --name "$BUILD_NAME" --source ..
 
-# Find 25% of the relevant tests to run for this change
+echo -e "\n# Find 25% of the relevant tests to run for this change"
```

Comment:
AIU, the `-e` option in this change like this one is pointless, because the shell is still replacing `\n` to LF (and the likes), so by the time the `echo` command sees it there's no backslash character left in its arguments.

Compare `echo -e \"` vs `echo \"` then contrast that with `echo -e \\\\` vs `echo \\\\`

I think we should remove all `-e` from echo commands

Reply:
Good catch! This was actually left in by accident: I was using `echo -e "\n ...` but switched to including simple `echo` statements to create newlines because it improved legibility. I missed this one when doing that refactor. 

However, it _did_ work as I expected (a newline was inserted). I believe that's because `-e` only enables interpretation of certain characters, including `\n`, but I don't know _how_ it gets around what you're describing...